wordpress-mobile/WordPress-iOS
# Jetpack Focus: Add analytics for static screens

Part of #20323 
Part of #20324 
Part of #20325

## Description
This PR adds event tracking for the "Moved to Jetpack" static screens.

Project spec: pcdRpT-22I-p2

## How to test


1. In `DashboardQuickActionsCardCell.swift`, replace `L72` with
```
            let movedToJetpackViewController = MovedToJetpackViewController(source: .stats)
            sourceController.navigationController?.pushViewController(movedToJetpackViewController, animated: true)
```
2. Build and run the WordPress app
3. On the My Site screen, switch to the dashboard tab
4. Tap on the Stats quick action cell (The small text cell at the top)
5. ✅ Tracked: `remove_static_poster_displayed <source: stats>`
6. Tap on the "Learn more" link
7. ✅ Tracked: `remove_static_poster_link_tapped <source: stats>`
8. Exit the webview and tap on the "Switch to the Jetpack app" button
9. ✅ Tracked: `remove_static_poster_get_jetpack_tapped <source: stats>`
10. Repeat the above steps for the `.reader` and `.notifications` sources, and make sure the correct source is captured

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
